changelogs/unreleased/gh-10709-luajit-fixes.md

```diff
@@ -0,0 +1,16 @@
+## bugfix/luajit
+
+Backported patches from the vanilla LuaJIT trunk (gh-10709). The following
+issues were fixed as part of this activity:
+
+* Fixed compilation of `getmetatable()` for `io` objects.
+* Fixed dirty reads from recorded `IR_NOP`.
+* Fixed fusing optimization across `table.clear()` or insertion of a new key.
+* Disabled FMA optimization on aarch64 to avoid incorrect results in floating
```

Comment:
aarch64 -- похоже на опечатку (arch64?)

Reply:
No, aarch64 is the 64-bit state introduced in the [Armv8-A architecture](https://en.wikipedia.org/wiki/ARM_architecture_family#Armv8-A). It may be written as AArch64, or aarch64, or arm64.